engine/src/main/java/org/terasology/rendering/dag/nodes/CopyImageToScreenNode.java

```diff
@@ -60,12 +65,15 @@ public void initialise() {
     public void process() {
         PerformanceMonitor.startActivity("rendering/copyImageToScreen");
         sceneFinalFbo.bindTexture(); // TODO: Convert to a StateChange
```

Comment:
Ideally, we should be able to. I tried converting it, but for some reason I could not get it to work (I think it was mid-PR, lots of broken files. I'll try again, and post an update!)